packages/docs/docs/core-api/synchronizer.md

```diff
@@ -176,3 +176,17 @@ Get the number of state tree leaves in a certain epoch.
 ```ts
 synchronizer.numStateTreeLeaves(epoch: number): Promise<number>
 ```
+
+## genSealedEpochProof
+
+Generate the sealed epoch proof. See [`sealEpoch`](../contracts-api/unirep-sol.md#sealepoch)
```

Comment:
Maybe add a note that this should only be used for small trees? Using snarkjs will be much slower than rapidsnark.